common/source/docs/common-tbs-rc.rst

```diff
@@ -34,7 +34,18 @@ See :ref:`common-crsf-telemetry` for information about telemetry data sent, disp
 ELRS Receivers
 ==============
 
-ELRS can be setup in the same manner as CRSF above , however, bit 13 of :ref:`RC_OPTIONS<RC_OPTIONS>` should be set to alter the baudrate from 416KBaud that CRSF uses, to 420KBaud that ELRS uses. As mentioned above, the UART selected must have DMA capability.
+ELRS can be setup in the same manner as CRSF above , however, bit 13 of :ref:`RC_OPTIONS<RC_OPTIONS>` should be set to alter the baudrate from 416KBaud that CRSF uses, to 420KBaud that ELRS uses. As mentioned above, the UART selected must have DMA capability. ELRS can also be configured to output MAVLink, enabling full bidirectional Mavlink telemetry and RC control over the same link. See :ref:`ELRS <common-rc-systems>` for more information.
+
+ELRS MAVLink Configuration
+--------------------------
+
+Instead of CRSF protocol, MAVLink protocol can be used. In this case, using SERIAL 4 for example:
+
+- Set :ref:`SERIAL4_PROTOCOL <SERIAL4_PROTOCOL>` = 23
+- Set :ref:SERIAL4_BBAUD <SERIAL4_BAUDL>` = 460
```

Comment:
Shouldn't this be set to 2 for Mavlink 2?
```suggestion
- Set :ref:`SERIAL4_PROTOCOL <SERIAL4_PROTOCOL>` = 2
- Set :ref:SERIAL4_BAUD <SERIAL4_BAUDL>` = 460
```

Reply:
yep..copy/paste error....thanks